csp-mixin/azureconfig.libsonnet

```diff
@@ -1,7 +1,7 @@
 {
   _config+:: {
     local this = self,
-    dashboardTags: ['azure'],
+    dashboardTags: ['azure-cloud-provider'],
```

Comment:
This will change the tags used by the other azure dashboards that are generated by this lib as well. We'll probably want/need to increment versions appropriately when we integrate it downstream.

Reply:
yeah, As you can see on next images, GCP and Azure use the same tag name.

<img width="1300" alt="Screenshot 2024-09-10 at 09 22 55" src="https://github.com/user-attachments/assets/da1505cf-c7e9-41a4-845c-4ca162250a79">

<img width="1281" alt="Screenshot 2024-09-10 at 09 22 48" src="https://github.com/user-attachments/assets/f69b57b4-3c5f-4892-a425-957e4973dbf6">